apps/condo/domains/ticket/components/TicketPropertyHintContent.tsx

```diff
@@ -0,0 +1,66 @@
+import { Typography } from 'antd'
```

Comment:
Why not in TicketPropertyHint subfolder?

Reply:
moved